feature-utils/poly-look/src/react-components/overlays/screen.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import "./screen.css";
+import "../../css/layout.css";
```

Comment:
Is there a reason `layout.css` in not imported in `css/index.js` but used here? I'm guessing we'll have more layout options in the future, but if the usage is currently "scoped" to `Screen` maybe you could move it to `screen.css`. This would be more about showing intention as in practice after the bundling it makes no difference, anybody can use the `standard-layout` class on any element and it will work.

Reply:
Yeah I'll add it to index.js. I don't want to scope it to screen, while this will be the only place it's used right now

Reply:
Now that we have it in index, we can safely remove the import here.